.azure_pipelines/performance.yaml

```diff
@@ -0,0 +1,41 @@
+schedules:
+- cron: 0 6 * * 5
```

Comment:
Do you have a quick estimation about the time costs for this perf_check pipeline?  Will the 0 6 * * 5 make our CI job crowed? How about put it in weekend? Like 0 0 * * 0, or 0 0 * * 6

Reply:
Roberta large takes the longest time to run which is around 2 hours https://aiinfra.visualstudio.com/Model%20optimization%20Toolkit/_build/results?buildId=347589&view=results. I set this pipeline to run on Friday morning to let the team can check this metric on the last workday of a week.